sechub-pds-solutions/gosec/05-stop-single-sechub-network-docker-compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+yamlFile=$1
```

Comment:
Is the intention of this script to be able to stop a variety of pds-solutions? Is the intention to have a generic `stop` script?

Reply:
I had something in mind, but now it's gone!